ContactsManagerBackend/ContactsManager/ContactsManagerWebApi/Controllers/PhonesController.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Web.Http;
+using AutoMapper;
+using ContactsManagerBL;
+using ContactsManagerWebApi.Models;
+using PhoneBL = ContactsManagerBL.Models.Phone;
+
+namespace ContactsManagerWebApi.Controllers
+{
+    [RoutePrefix("api/contacts/{contactId}/phones")]
+    public class PhonesController : ApiController
+    {
+        private PhonesManager PhonesManager { get; } = new PhonesManager();
+
+        [Route("")]
```

Comment:
You should specify HTTP verbs explicitly, with appropriate attributes, when using attribute routing.